plugins/com.google.cloud.tools.eclipse.appengine.login/.classpath

```diff
@@ -3,5 +3,6 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="lib" path="lib/google-oauth-client-1.22.0.jar"/>
```

Comment:
Have you actually added that jar in the lib directory in this pull request? I think you'll need to, or is maybe Github not showing it to me because it's a binary file?